server/src/main/java/io/crate/metadata/pgcatalog/PgTypeTable.java

```diff
@@ -44,29 +45,41 @@ public static SystemTable<PGType<?>> create() {
         return SystemTable.<PGType<?>>builder(IDENT)
             .add("oid", INTEGER, PGType::oid)
             .add("typname", STRING, PGType::typName)
-            .add("typdelim", STRING, PGType::typDelim)
-            .add("typelem", INTEGER, PGType::typElem)
+            .add("typnamespace", INTEGER, c -> TYPE_NAMESPACE_OID)
+            .add("typowner", INTEGER, c -> null)
             .add("typlen", SHORT, PGType::typeLen)
             .add("typbyval", BOOLEAN, c -> true)
             .add("typtype", STRING, PGType::type)
             .add("typcategory", STRING, PGType::typeCategory)
-            .add("typowner", INTEGER, c -> null)
             .add("typisdefined", BOOLEAN, c -> true)
+            .add("typdelim", STRING, PGType::typDelim)
             // Zero for non-composite types, otherwise should point
             // to the pg_class table entry.
             .add("typrelid", INTEGER, c -> 0)
-            .add("typndims", INTEGER, c -> 0)
-            .add("typcollation", INTEGER, c -> 0)
-            .add("typdefault", STRING, c -> null)
-            .add("typbasetype", INTEGER, c -> 0)
-            .add("typtypmod", INTEGER, c -> -1)
-            .add("typnamespace", INTEGER, c -> TYPE_NAMESPACE_OID)
+            .add("typsubscript", REGPROC, c -> REGPROC_ZERO)
+            .add("typelem", INTEGER, PGType::typElem)
             .add("typarray", INTEGER, PGType::typArray)
             .add("typinput", REGPROC, PGType::typInput)
             .add("typoutput", REGPROC, PGType::typOutput)
             .add("typreceive", REGPROC, PGType::typReceive)
             .add("typsend", REGPROC, PGType::typSend)
+            .add("typmodin", REGPROC, c -> REGPROC_ZERO)
+            .add("typmodout", REGPROC, c -> REGPROC_ZERO)
+            .add("typanalyze", REGPROC, c -> REGPROC_ZERO)
```

Comment:
I think we want to return 0 here, as from psql I see the following for this col and it seems it has to do with range types:
```
                typname                 |      typanalyze       
----------------------------------------+-----------------------
 tsvector                               | ts_typanalyze
 int4range                              | range_typanalyze
 numrange                               | range_typanalyze
 tsrange                                | range_typanalyze
 tstzrange                              | range_typanalyze
 daterange                              | range_typanalyze
 int8range                              | range_typanalyze
 int4multirange                         | multirange_typanalyze
 nummultirange                          | multirange_typanalyze
 tsmultirange                           | multirange_typanalyze
 tstzmultirange                         | multirange_typanalyze
 datemultirange                         | multirange_typanalyze
 int8multirange                         | multirange_typanalyze
```